corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -804,6 +808,16 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
             $.getScript(gridPolyfillPath);
 
             registerContinueListener(this, options);
+            this.handleSmallScreenChange(this.smallScreenEnabled);
+        },
+
+        handleSmallScreenChange: function (enabled) {
+            CaseTileListView.__super__.handleSmallScreenChange.apply(this, arguments);
+            if (enabled) {
+                $('#content-container').addClass('full-width');
+            } else if (!this.options.sidebarEnabled) {
```

Comment:
Do we need to handle the case where small screen is not enabled but sidebar is enabled?

Reply:
No, because that's already handled in `app.js`. This only checks if it's _not_ enabled, to make sure it's safe to remove the class.